realm/realm-library/src/androidTest/kotlin/io/realm/entities/DictionaryClass.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.entities
+
+import io.realm.Mixed
+import io.realm.RealmDictionary
+import io.realm.RealmList
+import io.realm.RealmObject
+
+open class DictionaryClass : RealmObject() {
+
+    var myMixedDictionary: RealmDictionary<Mixed>? = null
+    var myBooleanDictionary: RealmDictionary<Boolean>? = null
+    var myBooleanList: RealmList<Boolean>? = null
```

Comment:
Is it needed?

Reply:
It's just for helping me building the proxy, just for reference. I will delete it later.